cmd/minikube/cmd/start_flags.go

```diff
@@ -96,7 +96,7 @@ const (
 	waitTimeout             = "wait-timeout"
 	nativeSSH               = "native-ssh"
 	minUsableMem            = 1800 // Kubernetes (kubeadm) will not start with less
-	minRecommendedMem       = 1900 // Warn at no lower than existing configurations
+	minRecommendedMem       = 2048 // Warn at no lower than existing configurations
```

Comment:
We need to fix this, because the value is _also_ used for looking at the available memory _inside_ the VM...

So we should be using 2048 MiB on the "outside", but look for at least 1700 MiB on the "inside"

```go:
        // ControlPlaneNumCPU is the number of CPUs required on control-plane
        ControlPlaneNumCPU = 2

        // ControlPlaneMem is the number of megabytes of memory required on the control-plane
        // Below that amount of RAM running a stable control plane would be difficult.
        ControlPlaneMem = 1700
```

There is a force flag to run it it anyway, but we should do the complaint (the same as kubeadm)

Reply:
agree with anders, I think we should not bump this, but check fo the outter VM memory

Reply:
the focus of this pr was to eliminate tests flake due to insufficient memory errors - shall i just remove this change then and leave the changes in how much memory tests allocate?

Reply:
i've reverted this specific change as it's irrelevant for the specific issue it tries to solve for flake tests and should be instead addressed under a separate issue/pr